app/controllers/v6/assessments_controller.rb

```diff
@@ -99,6 +107,20 @@ def parse_capitals(capital_params)
       end
     end
 
+    def parse_main_home(property_params)
+      x = property_params.slice(:value, :outstanding_mortgage, :percentage_owned, :shared_with_housing_assoc, :subject_matter_of_dispute)
+      y = { value: 0.0, outstanding_mortgage: 0.0, percentage_owned: 0.0, shared_with_housing_assoc: false, subject_matter_of_dispute: nil, main_home: true }.merge(x)
+      Property.new(**y)
+    end
+
+    def parse_additional_properties(property_params)
+      property_params.map do |attrs|
+        x = attrs.slice(:value, :outstanding_mortgage, :percentage_owned, :shared_with_housing_assoc, :subject_matter_of_dispute)
+        y = { value: 0.0, outstanding_mortgage: 0.0, percentage_owned: 0.0, shared_with_housing_assoc: false, subject_matter_of_dispute: nil, main_home: false }.merge(x)
```

Comment:
It would maybe be nicer if the Property object was the thing that knew about default values, rather than the AssessmentController having to be explicit about what values to use if they're not explicitly provided.